docker-compose.yml

```diff
@@ -120,6 +128,7 @@ services:
       args:
         # Should match version pinned by Starboard
         - CLANG_VER=${CLANG_VER:-365097-f7e52fbd-8}
+        - COBALT_CLANG_VER=${COBALT_CLANG_VER:-17-init-8029-g27f27d15-3}
```

Comment:
I don't think we should be changing Clang version on an old branch - it's expected to be stable.